### ISSUE TITLE
ParallaxMusic Sample Won't run without DataMarket

### ISSUE THREAD

**Issue by mscherotter** [Microsoft/WindowsUIDevLabs#30]
The ParallaxMusic sample won't run and shows an "Authentication Failed" message because the RemoteDataSource is missing valid CLIENT_ID and CLIENT_SECRET values.  Please update the sample to not have these dependencies.

**Comment by JustinXinLiu** [comment]
Follow the instructions from this [link](https://msdn.microsoft.com/en-us/library/dn546686.aspx#subscribe) on how to obtain the `CLIENT_ID` and `CLIENT_SECRET`.

This is the [link](https://datamarket.azure.com/dataset/xboxmusic/xboxmusicplatform) to get started with the registration.

**Comment by pgills** [comment]
Closing. We've removed the remote data source with an inline one. We've received mixed feedback calling into a service. It's a barrier to get folks to get an API key, but at the same time show a more real world example when you have async data. For now we'll stick with local sources to get folks jump started on the API's and look into whether we can do some examples with real service integrations in the future. Thanks!
